hydromt_sfincs/utils.py

```diff
@@ -671,6 +671,14 @@ def read_sfincs_map_results(
         [var for var in ds_map.data_vars.keys() if (var in rm.values())]
     )
 
+    # support for older sfincs_map.nc files
+    # check if x,y dimensions are in the order y,x
+    if ds_map.xc.dims[0] != "y":
+        ds_map = ds_map.transpose(..., "y", "x")
+    # check if corner_x,corner_y dimensions are in the order corner_y,corner_x
+    if ds_map.corner_xc.dims[0] != "corner_y":
+        ds_map = ds_map.transpose(..., "corner_y", "corner_x")
```

Comment:
If all these coordinates are always present (which should be the case right?) we could basically simplify this.

```suggestion
    ds_map = ds_map.transpose(..., "y", "x", "corner_y", "corner_x")
```